openassessment/xblock/openassessmentblock.py

```diff
@@ -183,6 +192,12 @@ class OpenAssessmentBlock(
         help="The type of prompt. html or text"
     )
 
+    resubmissions_grace_period = String(
+        default="",
+        scope=Scope.content,
```

Comment:
nit: Why is this field `Scope.content` but `allow_learner_resubmissions` is `Scope.settings`? (I really should know the difference, but I don't, sorry..)

Reply:
I didn't know the difference either, but as I see in the [docs](https://edx.readthedocs.io/projects/xblock-tutorial/en/latest/concepts/fields.html#xblock-scope) for this case it would be more appropriate to use `Scope.settings` in both fields, so they will always be related to an instance.

Done: https://github.com/openedx/edx-ora2/commit/4d9a5f46083f98e469f6e7b5317abfb860f7b0fe